src/views/Auth/Login.jsx

```diff
@@ -26,23 +26,33 @@ export default function Login() {
     // from React Router to replace the current URL with the URL
     // we need to redirect to.
     // See https://v5.reactrouter.com/web/api/history for the appropriate method to use
+
```

Comment:
And here checking to make sure the user login was a success to get all of your test passing